dynamic-content/helpers.ts

```diff
@@ -58,8 +58,7 @@ function parseContent(acc: Record<string, string>, [key, value]) {
   if (key === "additionalDocs" && value) value = bulletsToArray(value);
   if (value && value.includes("<br>")) value = value.replaceAll("<br>", "\n");
   if (key === "civilPartnership") value = !!value;
-  if (key === "postal") value = !!value && value?.toLowerCase() !== "false";
-  if (key === "cniDelivery") value = !!value && value?.toLowerCase() === "true";
+  if (key === "postal" || key === "cniDelivery") value = !!value && value?.toLowerCase() !== "false";
```

Comment:
could you put this in a block/curly brace? it's hard to read run on lines like this.

seems like `!!value` is used a few times though? could we have something like `const hasValue = !!value`?